qemu/tests/cfg/balloon_in_use.cfg

```diff
@@ -1,43 +1,51 @@
 - balloon_in_use:
     type = driver_in_use
-    only Windows
     balloon = balloon0
     balloon_dev_devid = balloon0
     balloon_dev_add_bus = yes
     repeat_times = 5
-    ratio = 0.5
     wait_bg_time = 720
-    time_for_video = 600
     start_vm = yes
     kill_vm_on_error = yes
-    check_guest_bsod = yes
     run_bgstress = balloon_stress
-    stress_test = win_video_play
     bg_stress_run_flag = balloon_test
     set_bg_stress_flag = yes
-    free_mem_cmd = wmic os get FreePhysicalMemory
-    default_memory = ${mem}
     session_cmd_timeout = 240
     balloon_timeout = 240
```

Comment:
Is the timeout long enough for windows guest with 32G memory?

Reply:
From the history, no issues reported.

Reply:
Then it is ok to me, thanks!